<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QMainWindow" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartCafe Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_address">
      <property name="text">
       <string>ID / Address</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="combo_address">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_session_type">
      <property name="text">
       <string>Session Type</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QComboBox" name="combo_session_type">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_session_config">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../common/resources/common.qrc">
          <normaloff>:/img/settings.svg</normaloff>:/img/settings.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="button_connect">
        <property name="text">
         <string>Connect</string>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>305</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_aspia">
    <property name="title">
     <string notr="true">SmartCafe</string>
    </property>
    <addaction name="action_settings"/>
    <addaction name="action_clear_history"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_help"/>
    <addaction name="separator"/>
    <addaction name="action_check_for_updates"/>
    <addaction name="action_update_settings"/>
    <addaction name="separator"/>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menu_language">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu_aspia"/>
   <addaction name="menu_language"/>
   <addaction name="menu_help"/>
  </widget>
  <action name="action_help">
   <property name="text">
    <string>Online Help...</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_settings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_check_for_updates">
   <property name="text">
    <string>Check for updates...</string>
   </property>
  </action>
  <action name="action_update_settings">
   <property name="text">
    <string>Update Settings</string>
   </property>
  </action>
  <action name="action_clear_history">
   <property name="text">
    <string>Clear history</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../common/resources/common.qrc"/>
 </resources>
 <connections/>
</ui>
